pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -878,7 +878,10 @@ public Optional<String> selectBrokerForAssignment(final ServiceUnitId serviceUni
                     LoadManagerShared.applyNamespacePolicies(serviceUnit, policies, brokerCandidateCache,
                             getAvailableBrokers(),
                             brokerTopicLoadingPredicate);
-                    broker = placementStrategy.selectBroker(brokerCandidateCache, data, loadData, conf);
+                    Optional<String> brokerTmp = placementStrategy.selectBroker(brokerCandidateCache, data, loadData, conf);
```

Comment:
`src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java:[881] (sizes) LineLength: Line is longer than 120 characters (found 124).`
Please fix checkstyle violation

Reply:
Fixed.